package.json

```diff
@@ -93,6 +93,7 @@
     "theo": "8.0.0-beta.2",
     "tinycolor2": "^1.4.1",
     "xml": "^1.0.1",
+    "yaml-front-matter": "^4.0.0",
```

Comment:
Can we use https://www.npmjs.com/package/gray-matter instead? That's what we've been using on other `polaris-*` repos, might as well reuse the same thing. (also: it seems more popular)